src/main/java/org/example/repository/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.example.repository;
+
+import org.example.model.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
you should implement methods by yourself, not using JpaRepository
```suggestion
public interface BookRepository {
```